Libraries/ART/ARTNode.m

```diff
@@ -14,7 +14,16 @@ @implementation ARTNode
 - (void)insertReactSubview:(UIView *)subview atIndex:(NSInteger)atIndex
 {
   [super insertReactSubview:subview atIndex:atIndex];
+#if TARGET_OS_OSX // TODO(macOS ISS#2323203)
+  NSArray<__kindof NSView *> *subviews = self.subviews;
```

Comment:
This is done a few times. We should make a static line in RCTUIKit.h like RCTUIViewInsertSubView()

Reply:
We're doing this in libraries that don't consume the React lib (where RCTUIKit.h is defined) and I don't want to take on that dependency. This seems really minor to me and we don't do it often. Open to low-cost suggestions that we think FB would take though